ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -503,13 +511,40 @@ impl<'a> TraceBuilder<'a> {
         // This unwrap can't fail. If it does that means the tracer has given us a mappable block
         // that doesn't exist in the AOT module.
         self.create_trace_header(self.aot_mod.block(self.cp_block.as_ref().unwrap()))?;
-        let mut trace_iter = ta_iter.peekable();
+
         while let Some(tblk) = trace_iter.next() {
             match tblk {
                 Ok(b) => {
                     match self.lookup_aot_block(&b) {
                         Some(bid) => {
                             // MappedAOTBlock block
+
+                            if bid.is_entry() {
```

Comment:
I don't know if my eyes are wonky, but does `cargo fmt` make this `if` shift left one column?

Reply:
It's 4 spaces: time to go to spec savers? :p

Reply:
:facepalm: